### PR TITLE
Remove OpenSearch

### LINKED ISSUE #184: Remove Opensearch from CDK

### PR DESCRIPTION
**Pull request by sdhoyt**
# Change Summary

## Overview
This PR removes OpenSearch from the IMAP SDC

## New Dependencies
None

## New Files
None

## Deleted Files
All opensearch_utils files and opensearch_utils tests

## Updated Files
- indexer.py
- queries.py
- sds_data_manager_stack.py
- stack_builder.py
- pyproject.toml

All changes were only to remove any use of OpenSearch.

## Testing
- Updated sds_data_manager_stack unit tests
- cleared out the indexer unit test for now
- checked that it can be deployed without error
- checked that a file can be uploaded and indexer can run without error

### CODE REVIEW

File: sds_data_manager/lambda_code/SDSCode/indexer.py [hunk 1]
```diff
@@ -218,10 +162,7 @@ def lambda_handler(event, context):
         logger.info("Found the following metadata to index: " + str(metadata))
 
         # use the s3 path to file as the ID in opensearch
-        s3_path = os.path.join(os.environ["S3_DATA_BUCKET"], filename)
-        # create a document for the metadata and add it to the payload
-        opensearch_doc = Document(metadata_index, s3_path, Action.CREATE, metadata)
-        document_payload.add_documents(opensearch_doc)
+        os.path.join(os.environ["S3_DATA_BUCKET"], filename)
```
Comment (greglucas) [inline sds_data_manager/lambda_code/SDSCode/indexer.py]: Just remove this if it is unused. The comment above about opensearch too?

File: sds_data_manager/lambda_code/SDSCode/indexer.py [hunk 2]
```diff
@@ -232,15 +173,3 @@ def lambda_handler(event, context):
 
         # Write processing status data to DynamoDB.
```
Comment (greglucas) [inline sds_data_manager/lambda_code/SDSCode/indexer.py]: I think this was removed, so a rebase/merge of upstream/dev will remove some additional stuff for you.

File: tests/lambda_endpoints/test_indexer.py [hunk 3]
```diff
@@ -1,128 +1,18 @@
...
-
-from sds_data_manager.lambda_code.SDSCode import indexer
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.action import Action
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.client import Client
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.document import Document
-from sds_data_manager.lambda_code.SDSCode.opensearch_utils.index import Index
 
 
 @pytest.mark.network()
 class TestIndexer(unittest.TestCase):
```
Comment (greglucas) [inline tests/lambda_endpoints/test_indexer.py]: Can probably remove this entire file for now too? We have been moving over to the pytest fixture-based approach as well, so might want to change this file over to that by starting fresh.

**Review by greglucas — COMMENTED** [review]
The DynamoDB work was ripped out as well, so I think that may be the cause of some of the conflicts that need to be resolved.

**Review by tech3371 — APPROVED** [review]
yeah, it looks good. You will need to rebase with latest change but otherwise it looks good to me.

**Review by greglucas — APPROVED** [review]

**Review by bourque — APPROVED** [review]
There is something satisfying about removing so much code

**Pull request merged**